Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -192,7 +192,7 @@ def __init__(
         )
         self.folder_name = folder
         self.is_public_folder = is_public_folder
-        self.access_type = kwargs.get('access_type') or IMPERSONATION
+        self.access_type = kwargs.get('access_type', IMPERSONATION).lower()
```

Comment:
lets change to the following to be on the safe side:
```suggestion
        self.access_type = (kwargs.get('access_type', IMPERSONATION) or IMPERSONATION).lower()
```